custom/icds_reports/migrations/sql_templates/database_functions/insert_into_ccs_record_monthly.sql

```diff
@@ -175,7 +175,8 @@ BEGIN
     'swelling = agg.swelling, ' ||
     'blurred_vision = agg.blurred_vision, ' ||
     'convulsions = agg.convulsions, ' ||
-    'rupture = agg.rupture ' ||
+    'rupture = agg.rupture, ' ||
+    'home_visit_date = agg.latest_time_end_processed::DATE ' ||
```

Comment:
This will probably be a confusing name in the future. Home visit usually means an entire module, but this is only the birth preparedness form right?